applications/GeoMechanicsApplication/custom_processes/apply_constant_boundary_phreatic_surface_pressure_process.hpp

```diff
@@ -161,61 +124,18 @@ class ApplyConstantBoundaryPhreaticSurfacePressureProcess : public Process
     Vector3 mNormalVector;
     double mEqRHS;
 
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-private:
-
     void calculateEquationParameters()
     {
-        Vector3 v1;
-        Vector3 v2;
-        v1 = mSecondReferenceCoordinate - mFirstReferenceCoordinate;
-        v2 = mThirdReferenceCoordinate - mFirstReferenceCoordinate;
-
-        mNormalVector[0] = v1[1] * v2[2] - v1[2] * v2[1];
-        mNormalVector[1] = v1[2] * v2[0] - v1[0] * v2[2];
-        mNormalVector[2] = v1[0] * v2[1] - v1[1] * v2[0];
-
-        double sum=0;
-        for (unsigned int i=0; i < mNormalVector.size(); ++i)
-        {
-            sum += mNormalVector[i]*mNormalVector[i];
-        }
-        double norm = std::sqrt(sum);
-        if (!(norm > 0.0))
+        const Vector3 v1 = mSecondReferenceCoordinate - mFirstReferenceCoordinate;
+        const Vector3 v2 = mThirdReferenceCoordinate  - mFirstReferenceCoordinate;
+        mNormalVector = MathUtils<double>::CrossProduct(v1, v2);
```

Comment:
The template argument is optional and by default it is set to `double` (see line 60 of `math_utils.h`). My understanding is that at some point CIMNE will remove the class template parameter completely.

```suggestion
        mNormalVector = MathUtils<>::CrossProduct(v1, v2);
```

Reply:
Done